install/helm/agones/templates/extensions.yaml

```diff
@@ -49,6 +49,17 @@ spec:
   version: v1
 {{- end}}
 {{- if .Values.agones.registerWebhooks }}
+  template:

```

Comment:
Looks like this introduced error `error validating data: ValidationError(APIService.spec): unknown field "template" in io.k8s.kube-aggregator.pkg.apis.apiregistration.v1.APIServiceSpec`

Reply:
I've removed the `template/spec` as you suggested.